NDTensors/src/blocksparse/linearalgebra.jl

```diff
@@ -165,13 +165,32 @@ function LinearAlgebra.svd(T::BlockSparseMatrix{ElT}; kwargs...) where {ElT}
       Vb = copy(Vb)
     end
 
-    blockview(U, blockU) .= Ub
+    # <fermions>
+    sU = right_arrow_sign(uind, blockU[2])
+
+    if sU == -1
+      blockview(U, blockU) .= -Ub
+    else
+      blockview(U, blockU) .= Ub
+    end
+
     blockviewS = blockview(S, blockS)
     for i in 1:diaglength(Sb)
       setdiagindex!(blockviewS, getdiagindex(Sb, i), i)
     end
 
-    blockview(V, blockV) .= Vb
+    #<fermions>
+    sV = left_arrow_sign(vind, blockV[2])
+    # This sign (sVP) accounts for the fact that
+    # V is transposed, i.e. the index connecting to S
+    # is the second index:
+    sVP = block_parity(vind, blockV[2]) == 1 ? -1 : +1
```

Comment:
@emstoudenmire I think this line is the issue, I think it should just be:
```julia
    sVP = block_parity(vind, blockV[2])
```
When I change it to that, the ITensorGaussianMPS.jl tests pass.

Reply:
@emstoudenmire I added a fix to #1178, testing it out there. I want to merge it quickly since I'm worried that the bug I introduced in #1176 (which is now included in the latest registered version of ITensors, v0.3.40) is out there breaking user code, but if I merge the fix in #1178 and register a new version, the bug from this PR (#1174) will also cause issues. So if tests pass in #1178, I'll merge and register that, and then we can add more robust fermion tests in a future PR.

Reply:
Thanks for catching it! Yes, please go ahead with that plan